static/app/components/hookOrDefault.tsx

```diff
@@ -20,14 +18,6 @@ type Params<H extends HookName> = {
    * use React.Suspense and React.lazy to render the component.
    */
   defaultComponentPromise?: () => Promise<ReturnType<Hooks[H]>>;
-  /**
-   * Parameters to pass into the hook callback
-   */
-  params?: Parameters<Hooks[H]>;
```

Comment:
We never use this, and honestly we should just change these hooks so they're not functions.